recaptcha_enterprise/demosite/src/main/resources/application.properties

```diff
@@ -1,2 +1 @@
-spring.thymeleaf.prefix=file:src/main/resources/templates/
```

Comment:
We need this line!!!

Reply:
Local testing threw errors (guessing it's because of some local config). But console testing works. Hence restored.